miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -111,6 +115,17 @@ abstract class MiniAppMessageBridge {
         }
     }
 
+    private fun onShareContent(callbackId: String, jsonStr: String) {
+        try {
+            val callbackObj = Gson().fromJson(jsonStr, ShareInfoCallbackObj::class.java)
+
+            shareContent(callbackObj.param.shareInfo.content)
+            postValue(callbackId, "Share content successfully")
```

Comment:
"Content has been shared successfully." would be good? 🤔

Reply:
It's better to send this message after confirming the successfully execution from the HostApp. In that case, we need to rewrite the `abstract`  function as following:

```
abstract fun shareContent(
        content: String,
        callback: (String) -> Unit 
)
```

Therefore, we can invoke `callback` for passing a message of share result from HostApp.

Reply:
How about passing back some some constant value like `SUCCESS`?

Also, I agree with @khairul-alam-licon that we should have some way for the Host App to asynchronously send the callback success.

Reply:
Updated with status callback.